article-analysis-project/article-processing.js

```diff
@@ -0,0 +1,44 @@
+//Modules
+const unfluff = require('unfluff');
+const cheerio = require('cheerio');
+const request = require('request');
+//Variables and Consts
+const url = 'https://abcnews.go.com/Politics/wireStory/canceling-ivanka-trump-spurs-backlash-kansas-college-71174430?cid=clicksource_4380645_2_heads_hero_live_headlines_hed'
+
+//Main
+
+request(url, (error, response, html) => {
+    if (!error && response.statusCode == 200){
+        console.log(html);
```

Comment:
Usually want to remove debug console logs before merging; if it's an important log (e.g. it's a status report that you would want all users to see) that's one thing, but var dumps are usually best removed.